src/meta_schedule/feature_extractor/per_store_feature.cc

```diff
@@ -1313,6 +1313,7 @@ class PerStoreFeatureNode : public FeatureExtractorNode {
 
   Array<runtime::NDArray> ExtractFrom(const TuneContext& tune_context,
                                       const Array<MeasureCandidate>& candidates) {
+    TVMTimeScope(PerStoreFeatureNode::ExtractFrom);
```

Comment:
Hi Xiyou, most of the methods timed scopes here are not necessary and actually error prone in multi-threaded environment